source/includes/options-mongosqld.yaml

```diff
@@ -32,8 +32,34 @@ name: verbose
 directive: option
 aliases: -v
 description: |
-  Specifies that {{program}} should provide more detailed log output. Include
-  multiple times for more verbosity (e.g. ``-vvvvv``).
+  Specifies that {{program}} should provide more detailed log output.
+
+  The following table describes the information provided at each log
+  level:
+
+  .. list-table::
+     :widths: 10 20 10 60
+     :header-rows: 1
+
+     * - Letter Option
+       - Log Level
+       - Message Content
+     * - ``--quiet``
+       - None
+       - Log nothing.
+     * - No flag
+       - Default
+       - Log messages that notify user of basic :program:`mongosqld` events and state changes.
+     * - ``-v``
+       - Administrator
+       - Log messages that provie information to :program:`mongosqld` administrators.
+     * - ``-vv``
+       - Developer
+       - Log messages useful to MongoDB support and development.
+     * - ``-vvv``
```

Comment:
Trace was removed in the last iteration of the code review, but that change had apparently never made it back to the doc. I've updated the doc accordingly.

This means we can just remove this bullet point